experimental/sql/test/frontend/naming.ibis

```diff
@@ -0,0 +1,19 @@
+# RUN: rel_opt.py -f ibis %s | filecheck %s
```

Comment:
Can we rename this .py ? 
It wasn't clear to me this actually creates data out of thin air and injects it.

A typical mlir "opt" binary does not do that.

Reply:
Are you referring to the definition of the tables in rel_opt.py?

I think this is a discussion better placed in this [PR](https://github.com/google/iree-llvm-sandbox/pull/505), as this PR shows another drawback of our current filecheck approach.

Reply:
I changed parts of rel_opt in the [PR](https://github.com/google/iree-llvm-sandbox/pull/509). I think this addressed your comment here.

Reply:
Looks better but I would even go one step further in followup commits.
Let's land and iterate.